cloudinit/config/cc_reset_rmc.py

```diff
@@ -0,0 +1,167 @@
+# (c) Copyright IBM Corp. 2020 All Rights Reserved
+#
+# Author: Aman Kumar Sinha <amansi26@in.ibm.com>
+#
+# This file is part of cloud-init. See LICENSE file for license information.
+
+
+"""
+Reset RMC
+------------
+**Summary:** reset rsct node id
+
+Reset RMC module is IBM PowerVM Hypervisor specific
+
+Reliable Scalable Cluster Technology (RSCT) is a set of software components,
+that  together provide a comprehensive clustering environment (RAS features)
+for IBM PowerVM based virtual machines. RSCT includes the Resource monitoring
+and control (RMC) subsystem. RMC is a generalized framework used for managing,
+monitoring, and manipulating resources. RMC runs as a daemon process on
+individual machines and needs creation of unique node id and restarts
+during VM boot.
+More details refer
+https://www.ibm.com/support/knowledgecenter/en/SGVKBA_3.2/admin/bl503_ovrv.htm
+
+This module handles
+- creation of the unique RSCT node id to every instance/virtual machine
+  and ensure once set, it isn't changed subsequently by cloud-init.
+  In order to do so, it restarts RSCT service.
+
+Prerequisite of using this module is to install RSCT packages.
+
+**Internal name:** per instance
+
+**Supported distros:** rhel, sles and ubuntu
+
+"""
+
+import json
+import os
+import time
+
+from cloudinit.settings import PER_INSTANCE
+from cloudinit import util
+from cloudinit import subp
+
+frequency = PER_INSTANCE
+PIDOF = 'pidof'
+RMCCTRL = '/usr/sbin/rsct/bin/rmcctrl'
+RECFGCT = '/usr/sbin/rsct/install/bin/recfgct'
+SRCMSTR = '/sbin/srcmstr'
+
+NODE_ID_FILE = '/etc/ct_node_id'
+
+SRCMSTR_TIMEOUT_SECONDS = 30
+
+
+def handle(name, _cfg, _cloud, log, _args):
```

Comment:
we are generally speaking (i think... it'd be nice to have @blackboxsw or @OddBloke concur) using the 'log' that is passed in anymore.  That is mostly just an obsolete way of passing aroudn a logger.

The preferred way now is just:

```python
from cloudinit import logging
LOG = logging.getLogger(__name__)
...
LOG.debug("hi mom")
```

Reply:
@smoser  I referred the other modules, and followed the same convention in which logs are getting used under handle function.

Reply:
> @smoser I referred the other modules, and followed the same convention in which logs are getting used under handle function.

You're right, some existing modules *do* use the 'log' variable that is passed into handle, but newer ones do not (cc_apk_configure, cc_ubuntu_advantage, cc_snap, cc_zypper_add_repo).  We are moving away from that, as it really was never needed.